src/qt/utilitydialog.cpp

```diff
@@ -85,9 +84,6 @@ HelpMessageDialog::HelpMessageDialog(interfaces::Node& node, QWidget *parent, He
         std::string strUsage = node.helpMessage(HelpMessageMode::BITCOIN_QT);
         const bool showDebug = gArgs.GetBoolArg("-help-debug", false);
         strUsage += HelpMessageGroup(tr("UI Options:").toStdString());
-        if (showDebug) {
-            strUsage += HelpMessageOpt("-allowselfsignedrootcertificates", strprintf("Allow self signed root certificates (default: %u)", DEFAULT_SELFSIGNED_ROOTCERTS));
```

Comment:
missed `-rootcertificates`

Reply:
resolved in https://github.com/dashpay/dash/pull/4023/commits/47c10ea46bbe043b48c5c4c75b553fa7962420fc